examples/save_disp.py

```diff
@@ -1,67 +1,103 @@
 import os
+import sys
+from typing import List
 from pathlib import Path
 from packaging import version
+from argparse import ArgumentDefaultsHelpFormatter, ArgumentParser, Namespace
 
 import torch
 import numpy as np
 from PIL import Image
 
 
-def load_image(imfile):
-    """Utility function to load image files as tensors"""
-    img = Image.open(imfile)
+def load_image(im_path: Path, device: str = "cpu") -> torch.Tensor:
+    """Utility to load an image file as a torch tensor.
+
+    Args:
+        im_path (pathlib.Path): The path to the image file on the system.
+        device (str)    : The device on which the tensor should be loaded.
```

Comment:
Please adhere to the style guide.
```suggestion
        im_path: The path to the image file on the system.
        device : The device on which the tensor should be loaded.
```
Also, might as well simply write `image_path` instead of `im_path`, for readability.